backend/src/main/java/pl/edu/agh/awiteks_backend/services/PlantService.java

```diff
@@ -89,6 +89,17 @@ public void changeFavourite(int plantId, int userId) {
                 );
     }
 
+    public void changeReminders(int plantId, int userId) {
+        this.plantRepository
+                .findByIdAndUserId(plantId, userId)
+                .ifPresent(
+                        plant -> {
+                            plant.setSendReminders(!plant.isSendReminders());
```

Comment:
Dodałbym do rośliny metodę np. `changeSendReminders` i tam wkleił ten kod, bo to chyba lepiej będzie wyglądało
```java
private void changeSendReminders() {
     setSendReminders(!isSendReminders());   // Or do this: sendReminders = !sendReminders
}
```

Reply:
Done, ale pod nazwą toggle - change jest zbyt ogólne.